docs/tutorial/tutorial.md

```diff
@@ -0,0 +1,129 @@
+# Tutorial
+
+### Prerequisites
+
+To deploy wordpress-k8s charm, you will need a juju bootstrapped with any kubernetes controller.
+To see how to bootstrap your juju installation with microk8s, please refer to the documentation
+on microk8s [installation](https://juju.is/docs/olm/microk8s).
+
+### Setting up the environment
+
+To easily clean up the resources and to separate your workload from the contents of this tutorial,
+it is recommended to set up another environment by adding a new model with the following command.
+
+```
+juju add-model wordpress-tutorial
+```
+
+### Deploy wordpress-k8s charm
+
+Deployment of WordPress requires a database integration. A preferred method in juju is through the
+native juju relation. In this case, a `mysql` interface is required by the wordpress-k8s charm and
+hence, [`mysql-k8s`](https://charmhub.io/mysql-k8s) charm will be used.
+
+Start off by deploying the wordpress charm. By default it will deploy the latest stable release of
+the wordpress-k8s charm.
+
+```
+juju deploy wordpress-k8s
+```
+
+If you wish to try out the latest features of the wordpress-k8s charm you can set an additional
+parameter to deploy from a specific channel. For example, to deploy from the edge channel:
+
+```
+juju deploy wordpress-k8s --channel=edge
+```
+
+### Deploy and relate database
+
+#### Database relation
+
+The following commands deploys the mysql-k8s charm and relates wordpress-k8s charm through its mysql
+interface.
+
+```
+juju deploy mysql-k8s \
+    --config mysql-interface-user=wordpress \
+    --config mysql-interface-database=test-database
```

Comment:
Maybe call it `wordpress-tutorial-database`?

Reply:
Sounds good! changed other values too to note tutorial information.
i.e.
```
juju deploy mysql-k8s \
    --config mysql-interface-user=wordpress-tutorial-user \
    --config mysql-interface-database=wordpress-tutorial-database
```
```
microk8s kubectl run mysql -n wordpress-tutorial --image=mysql:latest \
--env="MYSQL_ROOT_PASSWORD=<strong-password>" \
--env=”MYSQL_DATABASE=wordpress-tutorial-database” \
--env=”MYSQL_USER=wordpress-tutorial-user” \
--env=”MYSQL_PASSWORD=<strong-password>
WORDPRESS_IP=microk8s kubectl get pod mysql --template '{{.status.podIP}}'
juju config wordpress-k8s \
db_host=$WORDPRESS_IP \
db_name=wordpress-tutorial-database \
db_user=wordpress-tutorial-user \
db_password=<strong-password>
```
Thank you for the suggestion!